bpf_filter.c

```diff
@@ -82,7 +82,8 @@ enum {
  *
  * Thanks to Ani Sinha <ani@arista.com> for providing initial implementation
  */
-#if defined(SKF_AD_VLAN_TAG_PRESENT)
+/* Npcap does not provide auxiliary data */
+#if defined(SKF_AD_VLAN_TAG_PRESENT) && !defined(NPCAP_AD_VLAN_TAG_PRESENT)
```

Comment:
Is it true in this context that so long as `SKF_AD_VLAN_TAG_PRESENT` is defined, `NPCAP_AD_VLAN_TAG_PRESENT` is never defined?

Reply:
As it currently stands, Npcap's `npcap-bpf.h` conditionally defines the same `SKF_AD_*` constants it did before (https://github.com/nmap/npcap/blob/a4e1690925da1c0086400a1dab7688567818ec5f/Common/npcap-bpf.h#L230-L238):

```c
/* Npcap SDK 1.15 defined these names instead, so they are preserved here for
 * compatibility and to simplify integration with existing Linux-targeted code.
 * However, it is preferred to use the NPCAP_AD_* prefix in new code. */
#ifndef SKF_AD_OFF
#define SKF_AD_OFF              NPCAP_AD_OFF
#define SKF_AD_VLAN_TAG         NPCAP_AD_VLAN_TAG
#define SKF_AD_VLAN_TAG_PRESENT NPCAP_AD_VLAN_TAG_PRESENT
#define SKF_AD_MAX              NPCAP_AD_MAX
#endif
```

So in a Npcap build, both constants are defined. We can still change this if necessary.